kafkaesk/consumer.py

```diff
@@ -300,6 +305,8 @@ async def _consume_batch(
         # Look for failures
         for task in done:
             record = futures[task]
+            tp = aiokafka.TopicPartition(record.topic, record.partition)
+            self._tp[tp] = record.offset + 1
```

Comment:
does adding 1 work when batch size is greater than 1? what if processing succeeds for only the Nth message from a batch of N messages, but experienced errors on all the others?

Reply:
yeah, good catch, changing this for max offset